src/guide/jbs-jdk-bug-system.md

```diff
@@ -219,6 +220,16 @@ Once the work on an issue has been completed the issues [Status]{.jbs-field} sho
 | [Approved]{.jbs-value} | [Closed]{.jbs-value} | Used as part of the [CSR process](https://wiki.openjdk.org/display/csr/Main). |
 | Challenge States | --- | [Exclude [Challenge]]{.jbs-value}, [Alternative Tests [Challenge]]{.jbs-value}, and [Reject [Challenge]]{.jbs-value} are only relevant within the context of the JCK Project. |
 
+### Closing issues as duplicates
+
+If the same issue is described in another JBS issue then close one against the other as [Closed]{.jbs-value}/[Duplicate]{.jbs-value}. In general the newer issue is closed as a [Duplicate]{.jbs-value} of the older one, but where the newer issue has a clearer description then doing it the other way round is ok as long as none of them has been [Fixed]{.jbs-value} already. If one of the issues has been [Fixed]{.jbs-value} the other one should be closed as a [Duplicate]{.jbs-value} of the [Fixed]{.jbs-value} issue.
```

Comment:
I agree that in general the newer bug should be closed in favor of the older, but it's not just about a clearer description.  It could be that the newer bug has more useful, up-to-date comments, or it somehow has more helpful attention/visibility.  For example, I would avoid closing a public bug as a duplicate of a confidential one.

Reply:
Agreed. Fixed. Thanks!